config/test.exs

```diff
@@ -57,7 +57,7 @@ config :oli, OliWeb.Endpoint,
   server: false
 
 # Config adapter for refreshing part_mapping
-config :oli, Oli.Publishing, refresh_adapter: Oli.PublishingTest.PartMappingMockAdapter
+config :oli, Oli.Publishing, refresh_adapter: Oli.Publishing.PartMappingRefreshSync
```

Comment:
Renamed this one since it's not used only in tests anymore.

Reply:
💯